packages/rss-db-collection/README.md

```diff
@@ -5,8 +5,9 @@ RSS/Atom feed collection for TanStack DB - sync data from RSS and Atom feeds wit
 ## Features
 
 - **📡 RSS & Atom Support**: Dedicated option creators for RSS 2.0 and Atom 1.0 feeds
-- **🔄 Automatic Polling**: Configurable polling intervals with intelligent error recovery and manual refresh capability
-- **✨ Deduplication**: Built-in deduplication based on feed item IDs/GUIDs
```

Comment:
Can we make this readme accessible from the website? Or if not or that's awkward (links breaking and whatever) let's move the docs with the rest of the docs and the readme can just link to Tanstack.com